packages/editor-ui/src/views/MfaSetupView.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+	<div :class="$style.container">
+		<div :class="$style.logoContainer">
+			<Logo />
+		</div>
+		<div :class="$style.headerContainer">
+			<n8n-heading tag="h2" size="xlarge" color="text-dark"
+				>Multi-factor Authentication Setup</n8n-heading
```

Comment:
reminder to please add i18n keys wherever possible

Reply:
@mutdmour I did. That screen MfaSetupView.view was a test and will be deleted. Now it's MfaSetupModal.vue. I did add all the i18n keys.